<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWidget</class>
 <widget class="QWidget" name="ConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>262</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="autoconnect">
     <property name="text">
      <string>Connect automatically with priority</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="prioritySpin">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>If the connection is set to autoconnect, connections with higher priority will be preferred. Defaults to 0.
The higher number means higher priority. An negative number can be used to indicate priority lower than the default.
           </string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="allUsers">
     <property name="text">
      <string>All users may connect to this network</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonPermissions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Edit advanced permissions for this connection</string>
     </property>
     <property name="whatsThis">
      <string>Fine-grained permissions editor for this connection. It lets you choose which users can activate/modify/delete this connection.</string>
     </property>
     <property name="text">
      <string>Advancedâ€¦</string>
     </property>
     <property name="icon">
      <iconset theme="preferences-desktop-user"/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="autoconnectVpn">
     <property name="text">
      <string>Automatically connect to VPN</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="vpnCombobox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="firewallZoneLabel">
     <property name="text">
      <string>Firewall zone:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>firewallZone</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="firewallZone">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="meteredLabel">
     <property name="text">
      <string>Metered:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>firewallZone</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="metered">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>NetworkManager indicates to applications whether the connection is metered and therefore
data usage should be restricted.</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
     <item>
      <property name="text">
       <string comment="Item in the Metered: list">Automatic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>autoconnect</tabstop>
  <tabstop>allUsers</tabstop>
  <tabstop>pushButtonPermissions</tabstop>
  <tabstop>autoconnectVpn</tabstop>
  <tabstop>vpnCombobox</tabstop>
  <tabstop>firewallZone</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>allUsers</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonPermissions</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoconnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>prioritySpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
